Add: members styles mobile

willm30 opened the pull request:
## Description

After I rebased the main feature branch on master, I needed to update some of the styles on the members page. This PR records that work.

**Changes** 🏗

* Changes go here

Resolves #3647

arrenv commented:
FYI, this is still in Draft

danbr reviewed (approved):
Looking very nice. Well done.  i'll approve, but have a question!!!
The only thing that i spotted is that the permissions are missing. Not sure if this is intentional? The permissions & reputations columns are pretty close so theres not much room to play with. How should this be handled?

With permissions
<img width="818" alt="Screenshot 2022-07-18 at 16 28 10" src="https://user-images.githubusercontent.com/582700/179522586-5996befd-67a6-4045-a7e6-7328e343d03d.png">

WithOUT permissions
<img width="432" alt="Screenshot 2022-07-18 at 16 28 21" src="https://user-images.githubusercontent.com/582700/179522600-177b7bce-84dc-4b29-a1f6-00340d30261d.png">

arrenv commented:
> Looking very nice. Well done. i'll approve, but have a question!!! The only thing that i spotted is that the permissions are missing. Not sure if this is intentional? The permissions & reputations columns are pretty close so theres not much room to play with. How should this be handled?

Yes, this is intentional. The permissions are within the `MemberPopover` after clicking on the avatar. We thought this was fine for a quick and dirty mobile implementation.

danbr commented:
@willm30 Please could you check that you committed any changes to: DomainDropdownItem.css.d.ts - (export const query700: string;)
Seems i modify the file when the dapp is complied on my system....... (maybe there is another explanation)

willm30 commented:
> @willm30 Please could you check that you committed any changes to: DomainDropdownItem.css.d.ts - (export const query700: string;) Seems i modify the file when the dapp is complied on my system....... (maybe there is another explanation)

@danbr Thanks for flagging. That line was not automatically added after carrying out the rebase on my machine, but have now added it and updated the feature branch 👍
